package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Delete this file from the repo root and apply these changes to the file in ./website